### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by alexander-lopez-s** [lab-brussels-1/All-about-trees-group4#10] [General]
# Review of our project

**Comment by sammou00** [comment]
Well done. Very nice job so far 

Some notes : 
![image](https://user-images.githubusercontent.com/67711948/159569613-03211d05-14dc-4100-89cb-b392df56096e.png)

**Comment by sammou00** [comment]
I think you need more user stories you still need images part , footer too .

**Reply by alexander-lopez-s** [reply]
Acknowledged!!! Thanks @samirm00

**Comment by sammou00** [comment]
Hello,

1- Very good job, well done , we all are very proud of you 

`Review` :

2- Very nice `user-personas`

3- Very nice `backlog` but we need more user stories , not just title and info also footer , pictures  

4- Very good  `development-strategy` 

5- No comments in `html` and `css` why ? 
see  group 3 discussion  [ group 3 ] (https://github.com/lab-brussels-1/All-About-Trees-Group-3/discussions/16)
